lib/inventory_sync/async/inventory_self_host_sync.rb

```diff
@@ -20,6 +20,10 @@ def create_facets
         results
       end
 
+      def rescue_strategy_for_self
+        Dynflow::Action::Rescue::Skip
+      end
```

Comment:
`InventoryHostsSync` plans `InventorySelfHostSync`, but they both run in concurrence. If both fail, the whole execution plan will pause itself, since one of the actions suggests to `Skip` and other to `Fail` and dynflow can't decide which is right.

We should either make them match or make `InventoryHostsSync` override the behavior by defining the strategy as `rescue_strategy` instead of `rescue_strategy_for_self`